scripts/check-cabal-files.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Quoting from https://github.com/input-output-hk/ouroboros-network/pull/4261#discussion_r1065554986:

> Also, the other scripts are in the `ci` subdirectory of `./scripts` instead of directly in it.

Can you move it there, too?

Reply:
Moved!